contentcuration/contentcuration/wsgi.py

```diff
@@ -13,21 +13,6 @@
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "contentcuration.settings")
 
-# Attach Python Cloud Debugger
```

Comment:
Will this affect our ability to trace issues? What did this do initially?

Reply:
@jayoshih we did have production debugging before, but it was never stable -- not sure if you remember all those struggles I had getting this to work reliably. With sentry the need for this is significantly diminished.